### PR TITLE
Bump

### PR DESCRIPTION
**Pull request by MaeIsBad**

### CODE REVIEW

**Review by cpiemontese — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -8,11 +8,16 @@ and this project adheres to
...
+---
+
+## [0.16.5] - 2024-07-10
+
 ### Security
 
 - Switched to using XChaCha20Poly1305 for the redis token cache encryption.
 
-This addresses a few medium severity security issues with the tokens
+This addresses a few medium severity security issues with the tokens. Note that
```
Comment (cpiemontese) [inline CHANGELOG.md:19]: Doesn't this imply a breaking change...?

**Review by cottinisimone — COMMENTED** [review]
People might be scared about `the token cache is incompatible with old versions of the library` is it worth it to explain that this will result only in more calls to auth0/okta?

File: CHANGELOG.md (hunk 1 above)
Comment (MaeIsBad) [reply CHANGELOG.md:19]: I wouldn't say so. You can't use previous versions of the library alongside the new one, but you can upgrade without any changes on your end

Comment (cpiemontese) [reply CHANGELOG.md:19]: I see, although I agree with Simone's comment below

**Comment by MaeIsBad** [comment]
@cottinisimone how do you feel about this? I tried rewording it a little

File: CHANGELOG.md [hunk 2]
```diff
@@ -8,11 +8,16 @@ and this project adheres to
...
+
+## [0.16.5] - 2024-07-10
+
 ### Security
 
 - Switched to using XChaCha20Poly1305 for the redis token cache encryption.
 
-This addresses a few medium severity security issues with the tokens
+This addresses a few medium severity security issues with the tokens.
+Note that this means that this, and future versions of the library cannot be used at the same time as older verisons.
```
Comment (cottinisimone) [inline CHANGELOG.md:20]:
```suggestion
Note that this means that this, and future versions of the library cannot be used at the same time as older versions.
```

**Review by cottinisimone — APPROVED** [review]

**Pull request merged**